test/fixtures/group-basic.json

```diff
@@ -18,10 +18,10 @@
   "output": {
     "user_id": "user-id",
     "time": 1388534400000,
-    "user_properties": {
+    "groups": {
```

Comment:
yeah but this isn't an order issue, it's changing the actual name of the properties @ladanazita